tests/arns.test.mjs

```diff
@@ -63,6 +63,22 @@ describe('ArNS', async () => {
     );
 
     const buyRecordData = JSON.parse(buyRecordResult.Messages[0].Data);
+    const buyRecordEvent = JSON.parse(buyRecordResult.Output.data.split('\n')[1]);
+    
+    // Fields based on timestamps will be unstable for testing, but verify their type
+    assert.equal("number", typeof buyRecordEvent.Timestamp);
+    assert.equal("number", typeof buyRecordEvent['Epoch-Index']);
+    assert.equal("number", typeof buyRecordEvent['Start-Timestamp']);
+    assert.equal("number", typeof buyRecordEvent['End-Timestamp']);
+    delete buyRecordEvent.Timestamp;
+    delete buyRecordEvent['Epoch-Index'];
+    delete buyRecordEvent['Start-Timestamp'];
+    delete buyRecordEvent['End-Timestamp'];
```

Comment:
these are all controllable in the test setup, likely don't need to delete and node does support things like `assert.isNumber()` if we don't care about specific values

Reply:
The values are just unstable in testing environment since they're based on the current timestamp. Thanks for the tip on isNumber